projects/ngx-maplibre-gl/src/lib/control/control.component.ts

```diff
@@ -13,14 +13,24 @@ import { MapService } from '../map/map.service';
 export class CustomControl implements IControl {
   constructor(private container: HTMLElement) {}
 
+  /**
+   * @returns HTMLElement
+   */
   onAdd() {
     return this.container;
   }
 
+  /**
+   * @returns HTMLElement
```

Comment:
I would suggest to update the typescript definition of the method instead of adding a comment.
i.e. `onRemove(): HTMLElement`

Reply:
I added the javascript comment for `npm run docs` not to fail. If I just do what you suggest, it will fail:

```
> ngx-maplibre-gl-srcs@18.0.0 docs
> typedoc

[warning] Failed to resolve link to "Map#removeControl" in comment for CustomControl.onRemove
[warning] Found 0 errors and 1 warnings
```

Reply:
Can you add "inherit doc" or something? It would be better in this case I think.
If not, then a more meaningful comment would be better...

Reply:
I tried but could not make it to work.